test/tests.test.js

```diff
@@ -8,7 +8,7 @@ const Fastify = require('fastify')
 const plugin = require('../')
 const semver = require('semver')
 
-const urlHost = '127.0.0.1'
+const urlHost = 'localhost'
```

Comment:
🤔 I wonder how this performs on Windows. Do the tests pass? If I recall correctly, Windows has resolution issues with `localhost` due to dual-stack IP.

Reply:
We are not currently running the test suite on windows, so I would be happy to land this.

Reply:
I can check on Windows later (currently virtually attending other sessions at OpenJS World Collaborators Summit).

Reply:
> We are not currently running the test suite on windows

I know. I'm not sure it is worth adding to the suite, either.

Reply:
then let's land this, shall we?